docs/Using Price Feeds/historical-price-data.html

```diff
@@ -31,7 +31,51 @@
 
 # Historical Rounds
 
+To get historical data, one needs to call the `getRoundData` [function](/docs/price-feeds-api-reference/#getrounddata) providing `roundId` as parameter.
 Data Feeds are updated in rounds. Rounds are identified by their `roundId`, which increases with each new round. The increase might not be monotonic. Knowing the `roundId` of a previous round allows contracts to consume historical price data.
+Technically, the `roundId` is computed as the following in the [proxy contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol):
+
+```solidity
+roundId = uint80((uint256(phaseId) << 64) | originalId);
+```
+
+where:
+
+- `phaseId`: is incremented each time the underlying aggregator implementation has been updated. More info on the relationship between _proxies_ and _aggregators_ can be found [here](/docs/architecture-decentralized-model/).
+- `originalId` : is the round id of the current underlying aggregator. The id starts at 1. 
+- `roundId`: computed round id. From the above formula, you can think of it as a large number containing `phaseId` and `originalId`.
+
+When querying the historical data, it is important to know when we hit the history of the underlying aggregator. Let's illustrate this with an example:
```

Comment:
```suggestion
When querying the historical data, it is important to know when we have reached the history of the underlying aggregator. Let's illustrate this with an example:
```

Not sure if this is the right wording

Reply:
ok will correct with next push